catroid/build.gradle

```diff
@@ -151,7 +151,7 @@ dependencies {
     compile fileTree(include: '*.jar', dir: 'src/main/libs-natives')
     androidTestCompile fileTree(include: '*.jar', dir: 'src/androidTest/libs')
     androidTestCompile 'com.jayway.android.robotium:robotium-solo:5.2.1'
-    androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.2'
+    androidTestCompile 'com.linkedin.dexmaker:dexmaker-mockito:1.5.1'
```

Comment:
well, if it's only for tests, it's ok. in the worst scenario we have to rewrite the test if this stops working for any reason.